scale/product/models.py

```diff
@@ -462,6 +516,33 @@ def upload_files(self, file_entries, input_file_ids, job_exe, workspace):
                         product.meta_data = props
                     product.center_point = geo_utils.get_center_point(geom)
 
+            # Add recipe info to product if available.
+            recipe_check = Recipe.objects.get_recipe_for_job(job_exe.job_id)
+            if recipe_check:
+                recipe_manager = RecipeManager()
+                recipe_info = recipe_manager.get_details(recipe_check.id)
+                product.recipe_id = recipe_check.id
+                product.recipe_type = recipe_info.recipe_type.name
+                product.recipe_job = job_exe.get_job_type_name()
+
+                job_manager = JobManager()
+                job_details = job_manager.get_details(job_exe.job_id)
+                product.job_output = str(job_details.outputs)
```

Comment:
It's going to take a bit of work to do job_output. I'll have to talk to you about it.